src/Integration.UnitTests/Binding/SolutionBindingOperationTests.cs

```diff
@@ -259,12 +251,21 @@ public void SolutionBindingOperation_Prepare()
             testSubject.Prepare(CancellationToken.None);
 
             // Assert
-            prepareCalledForBinder.Should().BeTrue("Expected to propagate the prepare call to binders");
```

Comment:
I will be able to test that the relevant Binders are called only in future commits, after I change the code to use IProjectBinder instead of ProjectBindingOperation.